app/org/sagebionetworks/bridge/validators/IdentifierUpdateValidator.java

```diff
@@ -44,9 +46,15 @@ public void validate(Object object, Errors errors) {
         int updateFields = 0;
         if (update.getPhoneUpdate() != null) {
             updateFields++;
+            if (!Phone.isValid(update.getPhoneUpdate())) {
+                errors.rejectValue("phoneUpdate", "does not appear to be a phone number");
+            }            
```

Comment:
Should we add tests to IdentifierUpdateValidatorTest to make sure we catch invalid emails and phone numbers?

Reply:
Added test of email validation (neither empty nor junk value pass). Invalid phone test exists.